posts/python_sql_engine.md

```diff
@@ -0,0 +1,187 @@
+# Writing a Python SQL Engine from Scratch
+[Toby Mao](https://www.linkedin.com/in/toby-mao/)
+
+## Introduction
```

Comment:
I think this is missing a summary of this post itself. That help readers know what to expect.